docs/models/Form.md

```diff
@@ -4,31 +4,32 @@
+| `tags`                         | No       | `List<string>`              | List of tags to be associated with the form                                                                                 | `new List<string>()`              |
+| `publishStartDate`             | No       | `DateTime?`                 | DateTime the form should become available                                                                                   | `null`                            |
+| `publishEndDate`               | No       | `DateTime?`                 | DateTime the form should become unavailable                                                                                 | `null`                            |
+| `unpublishedUserMessage`       | No       | `string`                    | The message to be shown to forms users when the form is not in the published time window                                    | `null`                            |
+| `cancelAction`                 | No       | `string`                    | Allowed values of "BACK", "URL", "CLOSE", "FORMS_LIBRARY"                                                                   | `"BACK"`                          |
+| `cancelRedirectUrl`            | No       | `string`                    | URL to be redirected too, only applies if `cancelAction` is "URL"                                                           | `null`                            |
+| `serverValidation`             | No       | `FormServerValidation`      | Optional configuration for form submission validation                                                                       | `null`                            |
+| `externalIdGenerationOnSubmit` | No       | `FormExternalIdGeneration`  | Optional configuration for generating externalId after serverValidation but before submission                               | `null`                            |
+| `personalisation`              | No       | `FormPersonalisation`       | Optional configuration for prefilling elements or generating new elements on form load                                      | `null`                            |
+| `approvalConfiguration`        | No       | `FormApprovalConfiguration` | Optional configuration for approvals                                                                                        | `null`                            |
+| `submissionTitle`              | No       | `string`                    | Optional configuration for the default submission title                                                                     | `null`                            |
+| `continueWithAutosave`         | No       | `boolean`                   | Whether or not viewing the form should auto-continue with autosave data when available rather than prompting the user title | `null`                            |
```

Comment:
```suggestion
| `continueWithAutosave`         | No       | `boolean`                   | Whether or not viewing the form should auto-continue with autosave data when available rather than prompting the user title | `false`                            |
```